### PR TITLE
fix: Add examples for sensitive fields to docs

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

**Review by iker-barriocanal — APPROVED** [review]

File: src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx [hunk 1]
```diff
@@ -27,7 +27,9 @@ With it enabled, Sentry will scrub the following:
   - github_token
   - privatekey
   - private_key
-- Any values whose keynames are contained in the list of additional fields in your Project Settings.
+- Values that contain strings in, or whose keynames are listed in, Sensitive Fields in your Project Settings.
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx:30]: Do you mean the "Additional Sensitive Fields" field in [Project] > Settings > Security & Privacy, or something else?
```suggestion
- Values that contain strings in, or whose keynames are listed in, [Project] > Settings > Security & Privacy in "Additional Sensitive Fields".
```

File: src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx [hunk 2]
```diff
@@ -27,7 +27,9 @@ With it enabled, Sentry will scrub the following:
...
+  - An entry in Sensitive Fields such as `mysekret` will cause removal of any field named `mysekret`, but also removes any field _value_ that contains `mysekret`. Sentry does this to protect against sensitive data leaking as part of structured data that has been sent as a single string to Sentry (such as, a JSON object that is stringified and embedded as JSON string in another JSON structure).
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx:31]:
```suggestion
  - An entry in "Additional Sensitive Fields" such as `mysekret`, for example, will cause the removal of any field named `mysekret`, but also removes any field _value_ that contains `mysekret`. Sentry does this to protect against sensitive data leaking as part of structured data that has been sent as a single string to Sentry (such as, a JSON object that is stringified and embedded as JSON string in another JSON structure).
```

File: src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx [hunk 3]
```diff
@@ -27,7 +27,9 @@ With it enabled, Sentry will scrub the following:
...
+  - As an extreme example where this behavior can become surprising, the string `"Unexpected error"` will be removed from events if there is an entry `exp` in Sensitive Fields.
```
Comment (imatwawana) [inline src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx:32]:
```suggestion
  - As an extreme example where this behavior can become surprising, the string `"Unexpected error"` will be removed from events if the entry `exp` is in "Additional Sensitive Fields".
```

**Review by imatwawana — COMMENTED** [review]
Made some style edits and updated/asked about a field name

File: src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx (hunk 1 above)
Comment (untitaker) [reply src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx]: yeah that 👍

**Pull request merged**